src/lib/holocene/checkbox.svelte

```diff
@@ -118,11 +118,17 @@
 
 <style lang="postcss">
   .checkbox {
-    @apply flex cursor-pointer select-none items-start gap-3 text-sm leading-[18px];
+    @apply flex cursor-pointer select-none items-start gap-3 text-sm leading-[18px] text-primary;
   }
 
   .checkbox.hoverable:hover .checkmark::before {
-    @apply absolute -left-2.5 -z-10 h-9 w-9 self-center rounded-full bg-interactive/20 content-[''];
+    @apply absolute -left-2.5 -z-10 h-9 w-9 self-center content-[''];
+  }
+
+  .checkbox.checked .checkmark {
+    &::after {
+      @apply bg-pink;
```

Comment:
Is this doing anything? Look like maybe it was left in by accident.

Reply:
Nice catch! Thanks Laura!